Dashboard/App.axaml.cs

```diff
@@ -27,12 +27,20 @@ public override void OnFrameworkInitializationCompleted()
             // Setup dependency injection
             var services = DependencyInjection.ConfigureServices();
 
-            desktop.MainWindow = new MainWindow
+          /*  desktop.MainWindow = new MainWindow
             {
                 DataContext = services.GetRequiredService<MainViewModel>()
+            };*/
+
+            desktop.MainWindow = new ConnectionWindow()
```

Comment:
Can you please merge in the main branch, and adjust your code to work with the connection tab?

Reply:
you didnt do this still